### PR TITLE
release note update for november release

### PR DESCRIPTION
**Pull request by bragi92**

### CODE REVIEW

**Review by rashmichandrashekar — DISMISSED** [review]

File: RELEASENOTES.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 # Azure Monitor Metrics for AKS clusters
 
 ## Pending
+
+## Release 11-02-2023
+* Linux image - `mcr.microsoft.com/azuremonitor/containerinsights/ciprod/prometheus-collector/images:6.8.1-main-<tbd>`
```
Comment (rashmichandrashekar) [inline RELEASENOTES.md:6]: will you be updating the version file later @bragi92 ?

Comment (bragi92) [reply RELEASENOTES.md:6]: aah, thanks @rashmichandrashekar ! For some reason I thought the version had already been upgraded. Added it in this PR.

Comment (gracewehner) [reply RELEASENOTES.md:6]: Looks like Vishwa had updated it yesterday for the new release with adding the new metric to the keeplist. I think 6.8.1 was right

Comment (bragi92) [reply RELEASENOTES.md:6]: Thanks Grace, I changed it back to 6.8.1

**Review by gracewehner — APPROVED** [review]

**Review by gracewehner — APPROVED** [review]

**Pull request merged**